lobby/src/test/java/org/triplea/lobby/server/db/UserControllerIntegrationTest.java

```diff
@@ -61,17 +61,16 @@ void testCreateDupe() {
 
   @Test
   void testLogin() {
-    final String password = md5Crypt(TestUserUtils.newUniqueTimestamp());
+    final String password = bcrypt(TestUserUtils.newUniqueTimestamp());
     final String user = newUserWithHash(password, Function.identity());
     controller.updateUser(
         user, generateEmailAddress(user), new HashedPassword(bcrypt(obfuscate(password))));
-    assertTrue(controller.login(user, new HashedPassword(password)));
     assertTrue(controller.login(user, new HashedPassword(obfuscate(password))));
   }
 
   @Test
   void testUpdate() throws Exception {
-    final String user = newUserWithMd5CryptHash();
+    final String user = newUserWithBCryptHash();
```

Comment:
I believe this test was in place to check if the legacy passwords were migrated correctly, so it can be simplified

Reply:
That original intent is not clear. I'm afraid you'll need to follow up on this if it is not simplified in other updates first.